pkg/restore/restore.go

```diff
@@ -388,6 +324,21 @@ type resourceClientKey struct {
 	namespace string
 }
 
+// getOrderedResources returns an ordered list of resource identifiers to restore, based on the provided resource
+// priorities and backup contents. The returned list begins with all of the prioritized resources (in order), and
+// appends to that an alphabetized list of all resources in the backup.
+func getOrderedResources(resourcePriorities []string, backupResources map[string]*archive.ResourceItems) []string {
+	// alphabetize resources in the backup
```

Comment:
Since we're alphabetizing now, does this mean Velero's restore order is now more determinate than it was before?

Reply:
we were alphabetizing before too (see https://github.com/vmware-tanzu/velero/pull/2248/files#diff-7e2ddda55b692c82f9fa5f89a3d05e54L149-L151) -- the only difference is now we're looking at what's in the backup, rather than what's in discovery